doc/src/vtr/server_mode/index.rst

```diff
@@ -202,7 +202,17 @@ Draw selected critical path elements example
 
   .. note:: If status is not 1, the field ***DATA*** contains error string.
 
+.. _ipa_client_for_testing_vpr_server_mode_functionality:
 
+IPA Client for Testing VPR Server Mode Functionality
```

Comment:
I'd change this title to "Interactive Path Analysis Client (IPA)"

Reply:
resolved ad6430b5797f59b987485ce609d4f0ab74fbdfb8